### ISSUE TITLE
CrossMediaManager.Android.NotificationIconResource not working

### ISSUE THREAD

**Issue by ricardoprestes** [martijn00/XamarinMediaManager#607]
## 🐛 Bug Report

The notification icon isn't showing same setting the NotificationIconResource.

### Expected behavior
Show my icon on the media notification message.

### Reproduction steps
```
CrossMediaManager.Current.Init(this);
CrossMediaManager.Android.NotificationIconResource = Resource.Drawable.my_icon;
```

### Configuration

**Version:** 0.8.8

**Platform:** 
- [ ] :iphone: iOS
- [X] :robot: Android
- [ ] :checkered_flag: WPF
- [ ] :earth_americas: UWP
- [ ] :apple: MacOS
- [ ] :tv: tvOS
- [x] :monkey: Xamarin.Forms

**Comment by EduardoPac** [comment]
I have the same problem

**Comment by ebetzler** [comment]
I had the same problem.  Determined that doing this during startup is too early. Seems that Android NotificationManager hasn't been properly initialized yet.  Doing the following was successful:

`            CrossMediaManager.Current.Init(this);
            CrossMediaManager.Current.PropertyChanged += Current_PropertyChanged;

        private void Current_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
        {
            System.Diagnostics.Debug.WriteLine("ANDROID MEDIA PROPERTY: " + e.PropertyName);

            switch (e.PropertyName)
            {
                case "MediaController":
                    {
                        CrossMediaManager.Android.NotificationIconResource = Resource.my_icon;
                        break;
                    }
            }
        }
`

**Comment by martijn00** [comment]
Should be fixed in 0.8.9
